.github/workflows/super_linter.yaml

```diff
@@ -54,10 +54,11 @@ jobs:
         # https://github.com/github/super-linter/blob/main/README.md
         env:
           VALIDATE_ALL_CODEBASE: false
-          VALIDATE_PYTHON_MYPY: false
           VALIDATE_DOCKERFILE_HADOLINT: false
+          VALIDATE_GITHUB_ACTIONS: false
           VALIDATE_MARKDOWN: false
+          VALIDATE_NATURAL_LANGUAGE: false
```

Comment:
What's the validation for and why is it being turned off?

Reply:
It's a natural language processor and it keeps borking the linter checks for things like "lock file" vs "lockfile" and "repository" vs "repo" - it's not catching anything useful.